.circleci/config.yml

```diff
@@ -129,12 +117,25 @@ jobs:
             source /tmp/venv/openfisca_core/bin/activate
             .circleci/publish-python-package.sh
 
+  check_version_and_changelog:
+    docker:
+      - image: python:3.6
+    steps:
+      - checkout
+
+      - run:
+          name: Check version number has been properly updated
+          command: |
+            git fetch
+            .circleci/is-version-number-acceptable.sh
+
 workflows:
   version: 2
   build_and_deploy:
     jobs:
       - build_python2
       - build_python3
+      - check_version_and_changelog
```

Comment:
In order to preserve behaviour, we need to add:

```yaml
- deploy_python2:
    requires:
      - build_python2
      - build_python3
      - check_version_and_changelog
```

Same for Python 3. Otherwise, if we remove the `master` filter, it will deploy a failing build nonetheless.